assets/js/components/help/HelpMenu.js

```diff
@@ -66,7 +66,7 @@ function HelpMenu( { children } ) {
 
 	const handleMenuSelected = useCallback( () => {
 		toggleMenu( false );
-	} );
+	}, [] );
```

Comment:
There's no reason for this to be a callback now technically but as per our tech decision, we want all handlers to use `useCallback` now

Reply:
`eslint-plugin-react-hooks` knows that `toggleMenu` comes from `useState` and thus does not need to be a dependency. Which is pretty cool